src/GeniusYield/Test/Privnet/Setup.hs

```diff
@@ -326,8 +335,8 @@ giveAda ctx addr = ctxRun ctx (ctxUserF ctx) $ do
 giveTokens :: Ctx -> GYAddress -> IO ()
 giveTokens ctx addr = ctxRun ctx (ctxUserF ctx) $ do
     txBody <- buildTxBody $
-        mustHaveOutput (mkGYTxOutNoDatum addr (valueSingleton (ctxGold ctx) 1_000_000)) <>
-        mustHaveOutput (mkGYTxOutNoDatum addr (valueSingleton (ctxIron ctx) 1_000_000))
+        mustHaveOutput (mkGYTxOutNoDatum addr (valueSingleton (ctxGold ctx) 10_000_000)) <>
```

Comment:
I think this was unnecessary but doesn't hurt to increase...?

I was debugging something but this didn't end up being the source.

Reply:
Yeah, it's fine.

Reply:
These tests had been a bit shaky in the past. Recently they seem to be more stable though, but I guess this won't hurt.